localisation/00_wc_sad.csv

```diff
@@ -705,3 +705,6 @@ whispers_option_stewardship;
 whispers_option_intrigue;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YIntrigue�!.;;;;;;;;;;;;;x
 whispers_option_learning;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YLearning�!.;;;;;;;;;;;;;x
 nick_the_corruptor;the Corruptor;;[From.Getderdie] Verderber[From.GetinFemale];;;;;;;;;;;x
+text_effect_qiraji_retinue;Enables �YSilithid Swarm�! retinue that is normally available to �YQiraji�! culture only.;;Aktiviert das �YSilithiden-Schwarm�!-Gefolge f�r die �YQiraji�!-Kultur.;;;;;;;;;;;x
+text_effect_naga_retinue;Enables �YRoyal Guard�! retinue that is normally available to �YNaga�! culture only.;;Aktiviert das �YK�nigswache�!-Gefolge f�r die �YNaga�!-Kultur.;;;;;;;;;;;x
+text_effect_titanforged_retinue;Enables �YTitan-forged Legion�! retinue that is normally available to �YTitan-forged�! culture group only.;;Aktiviert das �YTitanische Legion!-Gefolge f�r die �Ytitanische�! Kultur.;;;;;;;;;;;x
```

Comment:
I would remove dots and paint the retinue names green. Usually, effects don't have dots at the end.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77257027-34c80880-6c8b-11ea-8e60-e79ce03f3ad0.png)

</details>